src/lib/connector.ts

```diff
@@ -48,22 +49,6 @@ export async function connect(options: ConnectOptions): Promise<Connection> {
   }
 
   const signer = options.signer;
-  if (signer && signer.provider) {
-    // Set params with provider network info if not explicitly given in options
```

Comment:
Since a signer isn't required during connection anymore, moving the check to see if the wallet and tableland chains matched to a separate method lets us do this check when tokens are created, and when direct calls to the contract are made